src/fillTank.test.js

```diff
@@ -1,11 +1,100 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared ', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('should calculate full tank If the amount is not given', () => {
+    const customer = {
+      money: 3000, // customer account balance
+      vehicle: {
+        maxTankCapacity: 40, // fuel tank volume
+        fuelRemains: 0, // Remaining fuel in the tank
+      },
+    };
+
+    fillTank(customer, 30);
 
+    expect(customer.money).toBe(1800);
+    expect(customer.vehicle.fuelRemains).toBe(40);
   });
 
-  // write tests here
+  it(
+    'should calculate full tank If the amount is greater than the tank can use',
+    () => {
+      const customer = {
+        money: 3000, // customer account balance
+        vehicle: {
+          maxTankCapacity: 20, // fuel tank volume
+          fuelRemains: 5, // Remaining fuel in the tank
+        },
+      };
+
+      fillTank(customer, 50, 20);
+
+      expect(customer.money).toBe(2250);
+      expect(customer.vehicle.fuelRemains).toBe(20);
```

Comment:
The test description mentions calculating a full tank if the amount is greater than the tank can use, but the provided amount is 50, which is greater than the remaining capacity. Ensure the test logic and description are consistent.